lib/oppen/token.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+# Oppen
+module Oppen
+  # Token used in the oppen algorithm.
+  module Token
+    module BreakType
+      FITS = 0
+      INCONSISTENT = 1
+      CONSISTENT = 2
+    end
+
+    # String Token.
+    class String
+      attr_reader :value
+
+      def initialize(value)
+        @value = value
+      end
+
+      def length
+        @value.length
+      end
+    end
```

Comment:
Do we really need a wrapper on `String` ? Let's keep it for the moment because restricts the tokens to the set we define here, but we may want to remove it and just work with a normal string.